ui/app/AppLayouts/Chat/views/ChatMessagesView.qml

```diff
@@ -251,6 +251,13 @@ Item {
                     }
                 }
             }
+
+            onReplyDeleted: {
+                let item = chatLogView.itemAtIndex(messageIndex)
```

Comment:
This is very much undefined behavior; you are storing "state" inside a delegate by using an assignment. This will be invalidated as soon as the delegate is scrolled away and released by the ListView. When you scroll back, `delegate.replyMessage = null` will no longer be there

Reply:
@caybro I understand it, but no need to set it if delegate not exist. When you scroll back you it should be reassigned. If not exists - it should be assigned as well. This code just update visual delegates. If message is not exists (which is not true for now cos of status-go) it should be null as well. So basically, I use two conditions:

- If delegate in visual area and reply was deleted: need to find and delete reply from delegate
- If delegate was created: need to get original message and display it (if original message exists)

So it should be consistent in anytime.

Reply:
OK, let's see :)

Reply:
Anyway, I will fix it after https://github.com/status-im/status-desktop/issues/8369